makefile

```diff
@@ -191,6 +193,9 @@ endif
 EXAMPLES = $(basename $(wildcard examples/*.*))
 examples: $(EXAMPLES)
 ifneq ($(MINGW),ON)
+    ifeq ($(MSYS),ON)
+	    cp $(DYNLIB) test
+    endif
 # non-Windows: this task always runs them (note escaped $ to pass to bash)...
```

Comment:
So, your work changes the logic here. It seems like Windows-Msys uses the linux/osx cases in the makefile. So, the comment about non-Windows is actually wrong now, and might really confuse people. Could you explain this in comments? Thanks.

Reply:
Yes, using mingw compiler with msys is almost same as linux/osx cases.